nsot/models.py

```diff
@@ -1547,6 +1548,19 @@ class Circuit(Resource):
         )
     )
 
+    # This doesn't use the built-in SlugField since we're doing our own
+    # slugification (django.utils.text.slugify() is too agressive)
+    name_slug = models.CharField(
+        db_index=True,
+        editable=False,
+        max_length=255,
+        null=True,
+        unique=True,
```

Comment:
Using `null=True` on a CharField seems a little contradictory. It may be better to use `default=''`. See: https://docs.djangoproject.com/en/1.8/ref/models/fields/#null